### ISSUE TITLE
"Accept donations" does not stay checked even if I enter PayPal info

### ISSUE THREAD

**Issue by mjgardner** [metacpan/metacpan-web#2592]
I have tried entering my PayPal email address, PayPal.Me ID, and PayPal.Me URL, but none of these are saved after clicking "Save Profile".

**Comment by oalders** [comment]
Could you double check if three are any form errors on the page? They're not obvious in all cases. See https://github.com/metacpan/metacpan-web/issues/2378

**Comment by mjgardner** [comment]
> Could you double check if there are any form errors on the page? They're not obvious in all cases. See #2378

Nope, I get "Success / The profile has been updated." at the top of the page and there is no indication that there were any form errors.

**Comment by oalders** [comment]
I see your PayPal.Me URL in your profile: https://explorer.metacpan.org/?url=/author/MJGARDNER

**Comment by mjgardner** [comment]
Nice, so the only issue is seeing it on the edit profile page.

**Comment by ilmari** [comment]
https://metacpan.org/author/MJGARDNER has a "Donate" button, but clicking on it goes to https://www.paypal.com/donate/ which says "This organisation's page is broken."
